pkg/openstack/telemetry.go

```diff
@@ -227,11 +227,8 @@ func ReconcileTelemetry(ctx context.Context, instance *corev1beta1.OpenStackCont
 		instance.Spec.Telemetry.Template.Logging.DeepCopyInto(&telemetry.Spec.Logging)
 		instance.Spec.Telemetry.Template.MetricStorage.DeepCopyInto(&telemetry.Spec.MetricStorage)
 
-		// FIXME: need to switch telemetry operator enabled defaults to bool pointers to get around webhook defaulting issues
-		telemetry.Spec.Ceilometer.Enabled = instance.Spec.Telemetry.Template.Ceilometer.Enabled
-		telemetry.Spec.Autoscaling.Enabled = instance.Spec.Telemetry.Template.Autoscaling.Enabled
-		telemetry.Spec.Logging.Enabled = instance.Spec.Telemetry.Template.Logging.Enabled
-		telemetry.Spec.MetricStorage.Enabled = instance.Spec.Telemetry.Template.MetricStorage.Enabled
```

Comment:
We don't need to expclicitly copy MetricStorage.Enabled and Logging enabled as these values will get copied by the code on lines 227 and 228.